index.html

```diff
@@ -331,6 +334,18 @@
 		return x;
 	}
 }
+function pmOneToStr(x, p){
```

Comment:
pmOneToStr is cryptic, use toOutputString or prettyNumber instead.

Reply:
I changed name to 'toFixedWidth'